packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMImageUtils.m

```diff
@@ -0,0 +1,229 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FGMImageUtils.h"
+#import "Foundation/Foundation.h"
+
+static UIImage *scaleImage(UIImage *image, double scale);
+static UIImage *scaledImageWithScale(UIImage *image, CGFloat scale);
+static UIImage *scaledImageWithSize(UIImage *image, CGSize size);
+static UIImage *scaledImage(UIImage *image, NSNumber *width, NSNumber *height, CGFloat screenScale);
```

Comment:
The declaration comments for all of these functions should be here, on the declarations, rather than on the definitions below.